docs/querying/basics.md

```diff
@@ -202,7 +199,9 @@ PromQL supports line comments that start with `#`. Example:
 
         # This is a comment
 
-## Gotchas
+## Troubleshooting
```

Comment:
This section contains reference material about PromQL semantics, it does not make sense to be called troubleshooting.

Reply:
I changed it to Troubleshooting because it seems to be a section with information about how to solve common problems. "Gotchas" is vague and unclear, and certainly would not be commonly used in web searches. "Troubleshoot queries" would be.

Any countersuggestions to Troubleshooting?

Reply:
Maybe just remove the heading?

The staleness stuff is reference so belongs here, while the performance stuff is more guidey but it's probably the best place for it.

Reply:
Done